### PR TITLE
Unsigned integer constants

### PR DESCRIPTION
**Pull request by jenswi-linaro**
Updates following the coding guideline in https://github.com/OP-TEE/optee_docs/pull/117

### CODE REVIEW

**Comment by ruchi393** [comment]
`<Acked-by: Ruchika Gupta <ruchika.gupta@linaro.org>`

File: core/arch/arm/mm/core_mmu.c [hunk 1]
```diff
@@ -1622,7 +1622,7 @@ void core_mmu_map_region(struct mmu_partition *prtn, struct tee_mmap_region *mm)
 	vaddr_t vaddr = mm->va;
 	paddr_t paddr = mm->pa;
 	ssize_t size_left = mm->size;
-	int level;
+	unsigned int level;
```
Comment (jforissier) [inline core/arch/arm/mm/core_mmu.c:1625]: This should probably be a separate commit

**Review by jforissier — COMMENTED** [review]
>  There are a few cases of (1 << x) and (y << x), which are replaced or
> complemented by BIT(x) and SHIFT_U32(y, x) macros.
...which should probably be done is their own commit.

Overall:
`Acked-by: Jerome Forissier <jerome@forissier.org>`

File: core/arch/arm/mm/core_mmu.c (hunk 1 above)
Comment (jenswi-linaro) [reply core/arch/arm/mm/core_mmu.c:1625]: It is already in a separate commit, "core: core_mmu.[ch]: use U() for unsigned constants". Should I mention this change too in the commit message?

Comment (jforissier) [reply core/arch/arm/mm/core_mmu.c:1625]: If it is a direct consequence of using the `U()` constants then it's fine you can keep it like this and no need to mention it.

Comment (jenswi-linaro) [reply core/arch/arm/mm/core_mmu.c:1625]: Yes, there was a warning.

**Comment by jenswi-linaro** [comment]
Tags applied

**Comment by jforissier** [comment]
Not sure you noticed my comment about `BIT()` and `SHIFT_U32()` above (it was badly formatted)? Just a suggestion anyways.

Also please rebase, we now have a conflict.

**Comment by jenswi-linaro** [comment]
> Not sure you noticed my comment about `BIT()` and `SHIFT_U32()` above (it was badly formatted)? Just a suggestion anyways.

Sorry, missed that I'll fix.

> 
> Also please rebase, we now have a conflict.

I'll take care of that at the same time.

**Comment by jenswi-linaro** [comment]
Separated the BIT and SHIFT_U32 changes into separate patches. Rebased on master. I took the liberty to apply the tags to the two new commits too since the changes previously was part of the previously acked commits.

**Comment by jforissier** [comment]
@jenswi-linaro fine with me, thanks.
